src/test/java/bio/terra/service/resourcemanagement/google/GoogleProjectServiceTest.java

```diff
@@ -1,17 +1,26 @@
 package bio.terra.service.resourcemanagement.google;
 
 import bio.terra.common.category.Unit;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.junit4.SpringRunner;
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @RunWith(SpringRunner.class)
+@AutoConfigureMockMvc
+@SpringBootTest
```

Comment:
Once you add these, I think this becomes a connected test, not a unit test.

Reply:
Yeah, I was torn since it's only really when you uncomment...Maybe I'll actually just move this to an on-demand test, which is really more accurate